src/test/python/mmlsparktest/vw/test_vw_cb.py

```diff
@@ -0,0 +1,190 @@
+# Prepare training and test data.
+import os
+import unittest
+import tempfile
+import pyspark
+
+from mmlsparktest.spark import *
+
+from mmlspark.vw import VowpalWabbitContextualBandit
+from mmlspark.vw import VowpalWabbitFeaturizer
+from mmlspark.vw import VectorZipper
+
+from pyspark.ml.tuning import ParamGridBuilder
+from pyspark.sql.types import *
+from pyspark.ml import Pipeline
+from pyspark.ml.wrapper import *
+from pyspark.ml.common import inherit_doc, _java2py, _py2java
+from pyspark.sql.utils import AnalysisException
+
+def has_column(df, col):
+    try:
+        df[col]
+        return True
+    except AnalysisException:
+        return False
+
+class VowpalWabbitSpec(unittest.TestCase):
+    def get_data(self):
+        # create sample data
+        schema = StructType([
+            StructField("shared_text", StringType()),
+            StructField("action1", StringType()),
+            StructField("action2_feat1", StringType()),
+            StructField("action2_feat2", StringType()),
+            StructField("chosenAction", IntegerType()),
+            StructField("label", DoubleType()),
+            StructField("probability", DoubleType())
+        ])
+
+        data = pyspark.sql.SparkSession.builder.getOrCreate().createDataFrame([
```

Comment:
spark should already be availible to you because of the imports

Reply:
Fixed